WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -93,9 +95,8 @@ struct CartView: View {
                         }
                     }
                     .coordinateSpace(name: Constants.scrollViewCoordinateSpaceIdentifier)
-                    .onChange(of: cartViewModel.itemToScrollToWhenCartUpdated?.id) { _ in
-                        if viewModel.orderStage == .building,
-                           let itemToScrollTo = cartViewModel.itemToScrollToWhenCartUpdated?.id {
+                    .onChange(of: posModel.cart.first?.id) { itemToScrollTo in
```

Comment:
No change, this is the same logic. We add items to the top of the cart and scroll there.

It could perhaps be improved though; this means that we have the definition of where items are added in two places; the `insert` call, and here. I'll make a ticket to look at better ways in future.